### PR TITLE
Add Nozomi Networks Vantage Workflow #105

### PR DESCRIPTION
**Pull request by nicoloereni**
Signed-off-by: Nicolo Ereni <nicolo.ereni@nozominetworks.com>

### CODE REVIEW

**Comment by nicoloereni** [comment]
Hello @ChrisCollinsIBM, 
I've created this new PR because I did a mess with the old one...in this I did the changes asked .
Let me know if it's ok.
Thanks

File: Community Developed/NozomiNetworks/Vantage/NN-Vantage-Workflow.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+<Workflow name="NN-Vantage" version="1.0" minimumRecurrence="180" xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/Workflow/V1">
+  <Parameters>
+    <Parameter name="host" label="Host" required="true" />
+    <Parameter name="org" label="Organization" required="true" />
+    <Parameter name="key_name" label="Key Name" required="true" />
+    <Parameter name="key_token" label="Key Token" required="true" secret="true" />
+  </Parameters>
+
+  <Actions>
+    <Initialize path="/bookmark" value="0" />
```
Comment (ChrisCollinsIBM) [inline Community Developed/NozomiNetworks/Vantage/NN-Vantage-Workflow.xml:11]: Hey @nicoloereni, you mentioned in the last Pull Req that bookmark_assets as 0 was a good common use case for end users, but that they may want to adjust bookmark to a more recent timestamp to only pull in alerts of a certain age on the initial run. 

You had said you were going to add a comment regarding that somewhere but I don't see it here or in the readme, is this something you were still planning to add? 

If not no problem, I can merge as-is.

Comment (nicoloereni) [reply Community Developed/NozomiNetworks/Vantage/NN-Vantage-Workflow.xml:11]: Nice catch, I add a comment in the code and in the README.md 
Thanks

**Review by ChrisCollinsIBM — APPROVED** [review]
Looks great, thanks again for putting up with my picky reviews.

Thanks for your submission!

**Pull request merged**